keyboards/crkbd/rev1/common/keymaps/via/rules.mk

```diff
@@ -1,5 +1,13 @@
+SRC += ./lib/rgb_state_reader.c \
```

Comment:
Honestly, I would rather not include these files at all, and even remove them completely.  They a hold over from the older methods of rendering info on the OLEDs. 

Worse, is that because these are not the PROGMEM version of the functions, this uses up a LOT more memory. In fact, I've run into issues with the corne crashing because of memory overflow issues.   

As much of the OLED code should be PROGMEM as possible.  Which is why I wrote it that way, in the first place. 

And to be clear, this isn't a criticism, it's based off of issues that I had when converting over and implementing the code.

Reply:
oh, I understand your changed code in this file. 

I also didn't know what to do in regards to implementing OLED by this method for some time. The new rev1/common directory and refactoring about it may be a good idea. I'll probably revert this part.

Thanks.

Reply:
Well, revert everything but the `is_keyboard_master` part, since I did screw up the `is_master` part.